src/index.ts

```diff
@@ -1,15 +1,28 @@
 import * as ko from 'knockout';
 require('knockout.validation');
-import 'bootstrap/dist/css/bootstrap.min.css';
 import LeadTable from './pages/leadTable';
 
 ko.components.register('lead-table', LeadTable);
-class AppViewModel {
-  lead: { url: string }
+
+class ViewModel {
+  registros = [
```

Comment:
I don't see we are using this data in any place, please review if those are needed.